websocketOutportDriver/sender/webSocketSender.go

```diff
@@ -64,8 +64,8 @@ func NewWebSocketSender(args WebSocketSenderArgs) (*webSocketSender, error) {
 		server:                   args.Server,
 		counter:                  atomicCounter,
 		uint64ByteSliceConverter: args.Uint64ByteSliceConverter,
-		clients:                  make(map[string]*webSocketClient),
-		acknowledges:             make(map[string]map[uint64]struct{}),
+		clientsHolder:            NewWebsocketClientsHolder(),
+		acknowledges:             make(map[string]*websocketClientAcknowledgesHolder),
```

Comment:
Please refactor more: 
add this map also in the websocketClientAcknowledgesHolder (or other struct) Along with the mutex.

can add a TODO to refactor later

Reply:
done